test/functional/outputs/forward_to_kafka_test.go

```diff
@@ -0,0 +1,62 @@
+package outputs
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/kafka"
+	//"strings"
+	//. "github.com/openshift/cluster-logging-operator/test/matchers"
+)
+
+var _ = Describe("[LogForwarding][Kafka] Functional tests", func() {
+
+	var (
+		framework *functional.FluentdFunctionalFramework
+	)
+
+	BeforeEach(func() {
+		framework = functional.NewFluentdFunctionalFramework()
+	})
+	AfterEach(func() {
+		framework.Cleanup()
+	})
+
+	//What would be other Kafka specific values ?
+	setKafkaSpecValues := func(outspec *logging.OutputSpec) {
+		outspec.Kafka = &logging.Kafka{
+			Topic: kafka.AppLogsTopic,
+		}
+	}
+	join := func(
+		f1 func(spec *logging.OutputSpec),
+		f2 func(spec *logging.OutputSpec)) func(*logging.OutputSpec) {
+		return func(s *logging.OutputSpec) {
+			f1(s)
+			f2(s)
+		}
+	}
+
+	//	timestamp := "2013-03-28T14:36:03.243000+00:00"
+
+	Context("Application Logs", func() {
+		FIt("should send large message over Kafka", func() {
+			functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+				FromInput(logging.InputNameApplication).
+				ToOutputWithVisitor(join(setKafkaSpecValues, func(spec *logging.OutputSpec) {
+					//at this port fluent connects with kafka broker
+					spec.URL = "https://localhost:9093"
+				}), logging.OutputTypeKafka)
+			Expect(framework.Deploy()).To(BeNil())
+
+			var MaxLen uint64 = 40000
```

Comment:
I think that the value 40,000 had some meaning for syslog ... not sure there is a need for that in this firstKafaka test. you can use "WritesApplicationLogs" instead (this writes fix size of 100 bytes)